ONBOARDING.md

```diff
@@ -1,40 +1,67 @@
-<img src="https://nasaworldwind.github.io/css/images/nasa-logo.svg" height="100"/>
+<img src="https://cdn.earthdata.nasa.gov/eui/latest/docs/assets/ed-logos/meatball_hover_2x.png"/>
 
-# NASA World Wind Team
+# NASA WorldWind Team
 
-Welcome to the NASA World Wind team! This guide helps you get connected, explore the various resources you'll need as a
-team member, and start developing code for NASA World Wind.
+Welcome to the NASA WorldWind team! This guide helps you get connected, explore the various resources you'll need as a
+team member, and start developing code for NASA WorldWind.
 
-- [worldwind.arc.nasa.gov](https://worldwind.arc.nasa.gov) has all things World Wind in one place
+- [worldwind.arc.nasa.gov](https://worldwind.arc.nasa.gov) has all things WorldWind in one place
+- [Forum](https://forum.worldwindcentral.com) provides help and outreach to the WorldWind community
 - [GitHub](https://github.com/NASAWorldWind) hosts source code, issues, milestones, and releases
+- [Waffle.io](https://waffle.io/NASAWorldWind/NASAWorldWind.github.io) provides Agile project management
 - [Travis CI](https://travis-ci.org/NASAWorldWind) provides continuous integration and build automation
-- [World Wind Forum](https://forum.worldwindcentral.com) provides help and outreach to the World Wind community
+- [Bintray](https://bintray.com/nasaworldwind) hosts buid artifacts for Java, Android and ServerKit
+- [NPM](https://npmjs.com/org/nasaworldwind) hosts packages for WebWorldWind
 
-## Get Connected
+## Get Started
+
+Work with the WorldWind project manager to get connected with the WorldWind team. Be sure to have ...
 
 - [GitHub](https://github.com) account with two-factor authentication enabled
+- [Bintray](https://bintray.com/signup/oss) account (sign up with GitHub)
+- [NPM](https://www.npmjs.com/signup) account
 - [Skype](https://login.skype.com/account/signup-form) account
 - [JetBrains](https://account.jetbrains.com/login) account
-- [World Wind Forum](http://forum.worldwindcentral.com/register.php) account (NASA interns skip this step)
+- [Forum](http://forum.worldwindcentral.com/register.php) account (NASA interns skip this step)
 ... Set Avatar to NASA logo
 ... Set User Title to 'WW Team Member'
 ... Set Biography to 'NASA WorldWind Team Member'
```

Comment:
I believe two spaces before the dash space will work:
```
- Topic 1
   - Sub Topic A
   - Sub Topic B
- Topic 2
```

- Topic 1
  - Sub Topic A
  - Sub Topic B
- Topic 2